specification/storagesync/resource-manager/readme.nodejs.md

```diff
@@ -6,9 +6,9 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   azure-arm: true
-  package-name: azure-arm-storagesyncmanagement
-  package-version: 1.1.0
-  output-folder: $(node-sdks-folder)/lib/services/storagesyncmanagement
+  package-name: azure-arm-storagesync
+  package-version: 1.0.0
+  output-folder: $(node-sdks-folder)/lib/services/storagesyncManagement
```

Comment:
This is changing to match the reality in azure-sdk-for-node I trust?

Reply:
Yes. This package hasn't been published yet, so I changed it so that it follows our standard practices.